features/support/api_helpers.rb

```diff
@@ -122,15 +122,29 @@ def create_brief(framework_slug, lot_slug, user_id)
   brief_data = {
     updated_by: "functional tests"
   }
-  case lot_slug
-  when 'digital-specialists'
-    brief_data['briefs'] = Fixtures.digital_specialists_brief
-  when 'digital-outcomes'
-    brief_data['briefs'] = Fixtures.digital_outcomes_brief
-  when 'user-research-participants'
-    brief_data['briefs'] = Fixtures.user_research_participants_brief
-  else
-    puts 'Lot slug not recognised'
+  case framework_slug
+  when 'digital-outcomes-and-specialists'
+    case lot_slug
+    when 'digital-specialists'
+      brief_data['briefs'] = Fixtures.digital_specialists_brief
+    when 'digital-outcomes'
+      brief_data['briefs'] = Fixtures.digital_outcomes_brief
+    when 'user-research-participants'
+      brief_data['briefs'] = Fixtures.user_research_participants_brief
+    else
+      puts 'Lot slug not recognised'
```

Comment:
Could we have these raise exceptions rather than just print something out, or is the syntax more clunky?